src/cclib/parser/daltonparser.py

```diff
@@ -501,6 +515,9 @@ def extract(self, inputfile, line):
             self.set_attribute("charge", int(line.split()[-1]))
         if "@    Spin multiplicity and 2 M_S" in line:
             self.set_attribute("mult", int(line.split()[-2]))
+        # Dalton only has ROHF, no UHF
+            if self.mult != 1:
+                self.metadata["spintype"] = "ROHF"
```

Comment:
To me this also shows how `metadata` can be used as a staging ground for possible attributes, since `unrestricted` is perhaps something we might like to be an attribute(?). Anyway, I like this approach.